mantis-common/src/main/java/io/mantisrx/runtime/MachineDefinition.java

```diff
@@ -45,6 +46,7 @@ public MachineDefinition(@JsonProperty("cpuCores") double cpuCores,
         this.numPorts = Math.max(minPorts, numPorts);
     }
 
+    @VisibleForTesting
```

Comment:
nit: is this needed for public?

Reply:
unfortunately a lot of test classes are referring to this constructor from various packages.